x-pack/plugins/xpack_main/index.js

```diff
@@ -81,7 +82,19 @@ export const xpackMain = (kibana) => {
           }),
           type: 'string', // TODO: Any way of ensuring this is a valid email address?
           value: null
-        }
+        },
+        [CONFIG_ROLLUPS]: {
```

Comment:
I think this should be moved inside the [rollup plugin declaration](https://github.com/elastic/kibana/blob/master/x-pack/plugins/rollup/index.js) so that the settings are only enabled when the rollup plugin itself is enabled. as it currently is, the user will always see the settings if they have x-pack, even if they disable rollups. the `CONFIG_ROLLUPS ` constant can then live inside rollup directory as well, and the i18n keys be prefixed with `xpack.rollupJobs` instead 😄

Reply:
ah ok, that makes sense. thanks!

Reply:
@jen-huang updated!